files/en-us/web/api/readablestreambyobreader/index.html

```diff
@@ -34,7 +34,7 @@ <h2 id="Methods">Methods</h2>
  <dt>{{domxref("ReadableStreamBYOBReader.cancel()")}}</dt>
  <dd>Returns a {{jsref("Promise")}} that resolves when the stream is canceled. Calling this method signals a loss of interest in the stream by a consumer. The supplied <code>reason</code> argument will be given to the underlying source, which may or may not use it.</dd>
  <dt>{{domxref("ReadableStreamBYOBReader.read()")}}</dt>
- <dd>Returns a {{jsref("Promise")}} that resolves with an object indicating the state of the stream: either the net chuck in the stream or an indication that the stream is closed.</dd>
+ <dd>Returns a {{jsref("Promise")}} that resolves with an object indicating the state of the stream: either the chunk in the stream or an indication that the stream is closed.</dd>
```

Comment:
```suggestion
 <dd>Returns a {{jsref("Promise")}} that resolves with an object indicating the state of the stream: either the next chunk in the stream or an indication that the stream is closed.</dd>
```
“next chunk” is what was intended here. The relevant part of this spec is this:
https://streams.spec.whatwg.org/#default-reader-prototypea
> Returns a promise that allows access to the next chunk from the stream’s internal queue, if available.